packages/components/src/toggle-group-control/test/__snapshots__/index.js.snap

```diff
@@ -173,13 +173,6 @@ exports[`ToggleGroupControl should render correctly with icons 1`] = `
       id="toggle-group-control-1"
       role="radiogroup"
     >
-      <iframe
-        aria-hidden="true"
-        frameborder="0"
-        src="about:blank"
-        style="display: block; opacity: 0; position: absolute; top: 0px; left: 0px; height: 100%; width: 100%; overflow: hidden; pointer-events: none; z-index: -1;"
-        tabindex="-1"
-      />
```

Comment:
Should we not expect [this `div`](https://github.com/WordPress/gutenberg/blob/9398fcf74dbde80f24ce18b17d3ff01c69970127/packages/compose/src/hooks/use-resize-observer/index.tsx#L345-L359) to be rendered here instead of the `iframe` ?

(same for the `Flyout`'s snapshot)

Reply:
yes, I wasn't able to figure out why the snapshots didn't match here. My guess is that it has something to do with `act` and unit test weirdness. Any ideas?

Reply:
Not really. I don't think it's to do with `act`, because the snapshot is being created from a [very simple test](https://github.com/WordPress/gutenberg/blob/d5915916abc45e6682f4bdb70888aa41e98aa395/packages/components/src/flyout/test/index.js#L27-L30) — I also tried using Jest's fake timers, but nothing changed.

Maybe this is a symptom of an issue with the current implementation?

Reply:
Ok I've removed the global mock in my last commit. [db77508](https://github.com/WordPress/gutenberg/pull/40509/commits/db77508ab2affeb477496b8fa33951fd25272331)